plugins/snapshots/overlay/overlay.go

```diff
@@ -161,8 +177,16 @@ func NewSnapshotter(root string, opts ...Opt) (snapshots.Snapshotter, error) {
 		config.mountOptions = append(config.mountOptions, "index=off")
 	}
 
+	if config.enableQuota {
+		quotaSet, err = findQuotaSetter(root)
+		if err != nil {
+			return nil, err
```

Comment:
For file systems that do not support the quota mount option (ext4, etc.), do not return err here?

Reply:
"I'm not quite sure what you're trying to express, but an error has been returned and the process has ended here."

Reply:
I mean，do not return err here, and just print an warning log, and go on. so that to prevent report errors from on file systems that do not support quota.
like :
root@localhost.localdomain:/home/workspace$ df -h
Filesystem           Size  Used Avail Use% Mounted on
devtmpfs              30G     0   30G   0% /dev
tmpfs                 30G     0   30G   0% /dev/shm
tmpfs                 30G   18M   30G   1% /run
tmpfs                 30G     0   30G   0% /sys/fs/cgroup
/dev/mapper/cl-root   70G  9.9G   61G  15% /
/dev/vda1           1014M  257M  758M  26% /boot
/dev/mapper/cl-home  400G   16G  385G   4% /home
tmpfs                5.9G   32K  5.9G   1% /run/user/0
/dev/loop0           976M  6.1M  903M   1% /mnt/myext4
shm                   64M     0   64M   0% /mnt/myext4/state/io.containerd.grpc.v1.cri/sandboxes/26944b67479fc70fd12d1e74f675f11b349cd1490189948362c42db885ffb40e/shm
overlay              976M  6.1M  903M   1% /mnt/myext4/state/io.containerd.runtime.v2.task/k8s.io/26944b67479fc70fd12d1e74f675f11b349cd1490189948362c42db885ffb40e/rootfs
overlay              976M  6.1M  903M   1% /mnt/myext4/state/io.containerd.runtime.v2.task/k8s.io/52a1271cc6869d843aa87264986711fc8b69fc61680d57dcc065499f9546bd57/rootfs

Reply:
This doesn't seem to be the recommended approach. 

In principle, if configuration quotas are not supported, it should be reported and the process should exit. 

Additionally, it's possible to determine whether quota is supported when install and plan the containerd root directory in configure file.